src/i18n/en-actions.ts

```diff
@@ -34,8 +34,14 @@ const actionsMessageDescriptors = {
       ${ColonyExtendedActions.TokensUpdated} {Colony tokens were updated}
       ${ColonyExtendedActions.SafeRemoved} {Remove Safe}
       ${ColonyExtendedActions.SafeAdded} {Add Safe from {chainName}}
+      ${ColonyExtendedActions.SafeTransactionInitiated} {{safeTransactionTitle}}
       other {Generic action we don't have information about}
     }`,
+  [`action.title.${ColonyExtendedActions.SafeTransactionInitiated}.multipleTransactions`]: `Initiate Multiple Transactions from Safe Control`,
+  [`action.title.${ColonyExtendedActions.SafeTransactionInitiated}.transferFunds`]: `Transfer Funds from Safe Control`,
+  [`action.title.${ColonyExtendedActions.SafeTransactionInitiated}.transferNFT`]: `Transfer NFT from Safe Control`,
+  [`action.title.${ColonyExtendedActions.SafeTransactionInitiated}.contractInteraction`]: `Initiate Contract Interaction from Safe Control`,
+  [`action.title.${ColonyExtendedActions.SafeTransactionInitiated}.rawTransaction`]: `Initiate Raw Transaction from Safe Control`,
```

Comment:
Is this a leftover right? I see that the action titles are dynamic as they should be so I'm not seeing where these ones are used

Reply:
This is used in the actions list. I merged that pr yesterday so showing up here now.

Reply:
Oh, I think the title of the actions should be the same title we use in the action page, so the same title that the user inputs in the dialog.

That's at least the approach we use for every other action and I don't think there anything in the specs that indicate the contrary

Reply:
@ArmandoGraterol Have updated